site/content/docs/reference/active-directory-configuration.md

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
Maybe this per-field documentation should live in the API docs instead (or at least in addition to this)? That what we've done in the past. Then the other doc can include links to the API docs and/or the comments in the examples of the other doc can be copy/pasted from the API docs.